src/main/scala/edu/ie3/simona/model/participant/HpModel.scala

```diff
@@ -173,11 +224,19 @@ final case class HpModel(
       state: HpState,
```

Comment:
I agree with the alignment. It is really confusing. But we will get some colision e.g. around line 80 (scala doc of 'culculateActivePower') if there is no change the results of a tick before the current tick will still be current...

Reply:
Thinking longer on this, I guess changing this here into lastState would confuse why we are not updating the state here.

But I will have a look again on this later. And we may should have a short chat on this.

Reply:
Interesting. For the current implementation, the naming is wrong in `SystemParticipant` as well. I think nobody has ever noticed because `calculateActivePower` is not implemented by `StorageModel` and `EvcsModel` anyways. `ChpModel` and `BmModel` have not been tested within ParticipantAgents.
I'd suggest adapting the naming in `HpModel` at least, in order to make a transition to the new system participant model easier.

Reply:
Made an atempt, please have a look

Reply:
Yes, makes sense